samples/excel/89-preview-apis/chart-series-doughnutholesize.yaml

```diff
@@ -0,0 +1,138 @@
+id: excel-chart-series-doughnutholesize
+name: Chart series - doughnutHoleSize
+description: Set the doughnutHoleSize property in a series for a dough nut chart.
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#add-series").click(() => tryCatch(addSeries));
+        $("#delete-series").click(() => tryCatch(deleteSeries));
+
+        async function addSeries() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+
+                let seriesCollection = sheet.charts.getItemAt(0);
+                let rangeSelection = sheet.getRange("C2:C7");
+                let xRangeSelection = sheet.getRange("A1:A7");
+
+                // Add a series.
+                let newSeries = seriesCollection.series.add("Qtr2");
+                newSeries.setValues(rangeSelection);
+                newSeries.setXAxisValues(xRangeSelection);
+
+               // Set the size of the dough nut hole,
```

Comment:
Please add to the comment what the unit of measurement is. Are the holes measured in pixels?